GUIDELINES.md

```diff
@@ -136,3 +136,8 @@ In addition to the official Solidity Style Guide we have a number of other conve
     4. Declare the error in an extension if the error only happens in such extension or child contracts.
 
   * Custom error names should not be declared twice along the library to avoid duplicated identifier declarations when inheriting from multiple contracts.
+
+* It is important to follow a few rules around the use of Solidity overrides to avoid introducing unintended consequences due to the interaction with multiple inheritance:
```

Comment:
I think it's important is inferred because it's on the guidelines, and I'd rather write the guideline as a straight-forward rule:

```suggestion
* Solidity function overrides should follow the rules listed below in order to avoid introducing unintended consequences due to the interaction with multiple inheritance:
```